src/gui_tool/gui/general_gui.py

```diff
@@ -141,6 +143,16 @@ def play_video(self):
                     self.get_mode_handler().INSTRUCTIONS,
                 )
                 window.run()
+            elif self.key_mapper.consume("j"):
+                loc = TextPopup("Enter Index").run()
```

Comment:
I mostly meant for this commit to just be for you, since the sliders have the functionality for everyone else. If you still think it should be committed though, can you change the message to "Enter Index To Jump To, Or Close The Window To cancel"?